gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/JavaServiceDistributionExtension.groovy

```diff
@@ -21,6 +21,7 @@ import org.gradle.api.Project
 class JavaServiceDistributionExtension extends BaseDistributionExtension {
 
     private String mainClass
+    private String gcProfile = "throughput"
```

Comment:
do we actually want `throughput` as default?

Reply:
really - I just went with what's the current setup. For java 8 we might want to go with response-time.